oslabs-beta/apollo11
# Event Log, Event Details and Cache tab

**Feature:** 
Events and Cache tab

**Description:**
- Created the functionality to display events and the cache for each event on the events/cache tab

**Changes I Made:**
- Event log component - events objects passed down props from app. Map through the events on the events object and display a list of events and the operation name using material UI lists
- Event details component - display the operation and the response of the event using json react view
- Current cache component - display the current cache by mapping through the cache and displaying this using material UI list. Updated the button text to toggle between hide/show when clicked to change visibility of cache details component.
- Cache Details component - when a item in current cache is clicked on, the cache details will be displayed in cache details using react json view.

**How to Test:**
- Install the extension in a chrome browser
- Open the developer tools in the browser and click on the apollo 11 tab
- Navigate to a website using apollo client and interact with the website
- NOTE: You will need to refresh the website if you open the developer tools after the website loaded
- On the apollo 11 extension, click on the "Apollo Tab"
- You can click the three lines in the top left corner (hamburger menu) to expand the menu and see the text for each icon
- Click on events and the current cache section to view the functionality of these changes made in this PR
